src/main/java/mate/academy/spring/mapper/impl/response/UserResponseMapper.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.mapper.impl.response;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserResponseMapper implements DtoResponseMapper<UserResponseDto, User> {
+    @Override
+    public UserResponseDto toDto(User user) {
+        UserResponseDto userResponseDto = new UserResponseDto();
+        userResponseDto.setId(user.getId());
+        userResponseDto.setEmail(user.getEmail());
+        userResponseDto.setPassword(user.getPassword());
```

Comment:
Do we need to pass secure data to DTO?

<img width="386" alt="image" src="https://github.com/mate-academy/jv-spring-rest-web-practice/assets/72043323/9127b3ec-142a-4685-9136-f2121f3c93eb">